deb/build/debian/jenkins.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Jenkins Continuous Integration Server
+After=network.target
+
+[Service]
+Type=simple
+User=jenkins
+Group=jenkins
+SyslogIdentifier=jenkins
+UMask='0002'
```

Comment:
this must to be 0007 as a minimum (normally would be 0027) otherwise your Jenkins secrets could be read by users that should have no access to them.